tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/util/BFileUtil.java

```diff
@@ -132,4 +134,24 @@ public static String getQualifiedClassName(String orgName, String packageName, S
 
         return className;
     }
+
+    /**
+     * Delete all the files which are listed according to a given pattern.
+     *
+     * @param path    Path to the directory
+     * @param pattern Filter pattern
+     * @throws IOException Failed when deleting
+     */
+    public static void deleteAll(Path path, String pattern) throws IOException {
+        if (!Files.exists(path)) {
+            return;
+        }
+        List<Path> files = Files.find(path, Integer.MAX_VALUE, (p, attribute) ->
+                p.toString().contains(pattern)).collect(Collectors.toList());
+        if (files.size() > 0) {
```

Comment:
Do we need this `if` statement here? The `for each` won't run if the files.size() is less than zero.

Reply:
Fixed.